src/app/material.module.ts

```diff
@@ -0,0 +1,30 @@
+import { NgModule } from '@angular/core';
+import {
+  MatButtonModule, MatToolbarModule,
+  MatIconModule, MatListModule,
+  MatSidenavModule
+} from '@angular/material';
+import {LayoutModule} from '@angular/cdk/layout';
```

Comment:
follow the same code style as everywhere

Reply:
Fixed